File: Samples/CslaNet/cs/DeepData/DeepData/Form1.cs
using System;
using System.Windows.Forms;
using DeepData.Library;

namespace DeepData
{

	public partial class Form1 : Form
	{
		///<summary>
		///</summary>
		public Form1()
		{
			InitializeComponent();
		}

		private void OrderBindingNavigatorSaveItem_Click(object sender, EventArgs e)
		{
			Validate();
		}

		private void Form1_Load(object sender, EventArgs e)
		{
			OrderListBindingSource.DataSource = OrderList.GetList();
		}
	}
}

File: Samples/CslaNet/vb/DeepData/DeepData/Form1.vb
Public Class Form1

  Private Sub OrderBindingNavigatorSaveItem_Click(ByVal sender As System.Object, ByVal e As System.EventArgs)

    Me.Validate()

  End Sub

  Private Sub Form1_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load

    Me.OrderListBindingSource.DataSource = OrderList.GetList

  End Sub

End Class
